spring-cloud-zuul-ratelimit-core/src/main/java/com/marcosbarbero/cloud/autoconfigure/zuul/ratelimit/config/Rate.java

```diff
@@ -32,13 +36,27 @@
  * @author Marcos Barbero
  */
 @Data
+@Entity
 @NoArgsConstructor
 @AllArgsConstructor
+@Table(name = "rate_table")
```

Comment:
I don't really like to add prefix/suffix such as `_table` and `_column` to database objects.
Maybe just `rate_limit` as table name will be ok, also the column names just `key`, `limit` and so on will be ok.

Don't you think so?